WcaOnRails/db/migrate/20211126105123_update_preferred_formats_again.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class UpdatePreferredFormatsAgain < ActiveRecord::Migration[6.0]
+  def change
+    PreferredFormat.where(event_id: "222", format_id: "1").delete_all
```

Comment:
More of a general question for WST: what's the difference between `.delete_all` and `.destroy_all` here?

Reply:
There is a small paragraph about this in [`destroy_all`'s documentation](https://apidock.com/rails/ActiveRecord/Relation/destroy_all), but basically `delete_all` just fires the sql query, whereas `destroy_all` runs all the callbacks and destroys associated record if any.
In the case of `PreferredFormat` I think delete is just fine!